src/main/java/pl/llp/aircasting/networking/schema/ExportSession.java

```diff
@@ -0,0 +1,5 @@
+package pl.llp.aircasting.networking.schema;
+
+public class ExportSession
+{
+}
```

Comment:
If it's going to have an empty body, I would keep curly braces on the same line, i.e. `ExportSession {}`